calico-cloud/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -53,7 +53,7 @@ Vulnerability detection consists of these steps:
 
 Complete the following steps for each managed cluster you want enabled with the cluster scanner:
 
-1. Modify the `imageassurance` installation resource.
+1. Modify the [Image Assurance](../reference/installation/ia-api.mdx#image-assurance.operator.tigera.io/v1.ImageAssuranceSpec) installation resource.
```

Comment:
Need to update `ia-api.mdx` to include the new CRD changes.

Reply:
Got it, we'll make sure that runs at release time.